Packs/SecureWorks/Integrations/TaegisXDR/TaegisXDR.py

```diff
@@ -377,6 +396,84 @@ def fetch_alerts_command(client: Client, env: str, args=None):
     return results
 
 
+def fetch_assets_command(client: Client, env: str, args=None):
+    page = int(args.get("page", 0))
+    page_size = int(args.get("page_size", 10))
```

Comment:
```suggestion
    page = arg_to_number(args.get("page", 0))
    page_size = arg_to_number(args.get("page_size", 10))
```
Almost the same, just in the arg_to_number command we handle some edge cases :)

Reply:
try this:
    page = arg_to_number(args.get("page")) or 0
    page_size = arg_to_number(args.get("page_size")) or 10

Reply:
Gotcha. That took care of it. Thinks for the help.